browser/components/BoardShowPage/Card/CardModal.js

```diff
@@ -45,6 +49,20 @@ export default class CardModal extends Component {
     this.setState({showingLabelMenu: false})
   }
 
+  onCreateComment(comment) {
+    $.ajax({
+      method: 'post',
+      url: `/api/comments/card/${this.props.card.id}/board/${this.props.board.id}`,
+      contentType: 'application/json; charset=utf-8',
+      dataType: 'json',
+      data: JSON.stringify({comment: comment})
+    }).then(() => {
+      boardStore.reload()
+    }).catch(error => {
+      console.error(error)
```

Comment:
you should consider updating this components state when this error occures so you can display an error to the user and not just to developers in the console